testgrid/config/config.yaml

```diff
@@ -1806,12 +1806,16 @@ dashboards:
   dashboard_tab:
   - name: gke-cvm-1.6-cvm-master-upgrade-master
     test_group_name: ci-kubernetes-e2e-gke-cvm-stable-cvm-master-upgrade-master
+    description: 'Upgrade master only, in gke, from container_vm master-2 to container_vm master'
```

Comment:
currently master = 1.8, master-1 = 1.7, master-2 = 1.6... eh that's my laziness, so later on when master=1.9, master-1 will be 1.8